lib/backend/k8s/conversion/conversion_test.go

```diff
@@ -410,12 +391,12 @@ var _ = Describe("Test NetworkPolicy conversion", func() {
 		By("parsing the policy", func() {
 			pol, err = c.K8sNetworkPolicyToCalico(&np)
 			Expect(err).NotTo(HaveOccurred())
-			Expect(pol.Key.(model.ResourceKey).Name).To(Equal("knp.default.default.testPolicy"))
+			Expect(pol.Key.(model.ResourceKey).Name).To(Equal("knp.default.testPolicy"))
```

Comment:
We should add in an assertion on the Namespace being "default" as well.

Reply:
And related, make sure we have a test that checks the updateprocessor correctly namespaces the selectors when given a NetworkPolicy (if we don't already)

Reply:
Good call!  Uncovered a conversion bug.

Reply:
@robbrockbank Did you add this test? I don't see it.

Reply:
Oh it's way below. ok.